core/src/main/java/org/apache/accumulo/core/conf/ClientProperty.java

```diff
@@ -74,6 +74,17 @@
           + " use the table's durability setting. ",
       "2.0.0", false),
 
+  // ConditionalWriter
+  CONDITIONAL_WRITER_TIMEOUT_MAX("conditional.writer.timeout.max", "0", PropertyType.TIMEDURATION,
+      "Maximum amount of time an unresponsive server will be re-tried.", "2.0.0", false),
```

Comment:
Should document the behavior of zero, it seems to mean no timeout.

Reply:
Where do you think it should be documented? A comment?

Reply:
I was thinking of adding a sentence to `"Maximum amount of time an unresponsive server will be re-tried."`